sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/mongodb/MongoDbTable.java

```diff
@@ -17,12 +17,20 @@
  */
 package org.apache.beam.sdk.extensions.sql.meta.provider.mongodb;
 
+import static org.apache.beam.vendor.calcite.v1_20_0.org.apache.calcite.sql.SqlKind.AND;
+import static org.apache.beam.vendor.calcite.v1_20_0.org.apache.calcite.sql.SqlKind.COMPARISON;
+import static org.apache.beam.vendor.calcite.v1_20_0.org.apache.calcite.sql.SqlKind.OR;
 import static org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Preconditions.checkArgument;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
New precommit failure is indicating you should use vendored guava here

Reply:
Updated to vendored import. Thanks!